packages/app-mobile/components/voiceTyping/VoiceTypingDialog.tsx

```diff
@@ -142,22 +168,44 @@ const VoiceTypingDialog: React.FC<Props> = props => {
 		return <Text variant='labelSmall'>{preview}</Text>;
 	};
 
-	const headerAndStatus = <Text variant='bodyMedium'>{`${_('Voice typing...')}\n${renderContent()}`}</Text>;
 	return (
-		<Banner
-			visible={true}
-			icon={renderIcon()}
-			actions={[
-				{
-					label: _('Done'),
-					onPress: onDismiss,
-				},
-			]}
-		>
-			{headerAndStatus}
-			<Text>{'\n'}</Text>
-			{renderPreview()}
-		</Banner>
+		<Surface>
+			<View style={styles.container}>
+				<View style={styles.contentWrapper}>
+					<View style={styles.iconWrapper}>
+						<Icon source={renderIcon()} size={40}/>
+					</View>
+					<View style={styles.content}>
+						<AccessibleView
+							// Auto-focus
+							refocusCounter={1}
+							aria-live='polite'
+							role='heading'
+						>
+							<Text variant='bodyMedium'>
+								{_('Voice typing...')}
+							</Text>
+						</AccessibleView>
+						<Text
+							variant='bodyMedium'
+							// role="status" might fit better here. However, react-native
+							// doesn't seem to support it.
+							role='alert'
```

Comment:
See the list of [supported `role`s](https://reactnative.dev/docs/accessibility#role).